tests/App/AppKernel.php

```diff
@@ -91,6 +100,6 @@ protected function configureContainer(ContainerBuilder $containerBuilder, Loader
 
     private function getBaseDir(): string
     {
-        return sys_get_temp_dir().'/sonata-page-bundle/var/';
+        return sys_get_temp_dir().'/sonata-page-bundle---5353535a/var/';
```

Comment:
this should not be changed

Reply:
I know about this. I must change change this param each time I make functional test. I dont change it to correct becose I had some problem. @greg0ire  fix it.